test/api/v01/output_test.rb

```diff
@@ -192,7 +192,7 @@ def test_files_generated
     OptimizerWrapper.config[:debug][:output_clusters] = true
     OptimizerWrapper.config[:debug][:output_schedule] = true
 
-    vrp = TestHelper.load_vrp(self, fixture_file: 'instance_clustered')
```

Comment:
the method name `load_vrps` is implicitely expected to return an array. Assigning a simple `vrp` seems odd.

I would prefer to keep both `load_vrp` and `load_vrps` but to factorize their codes.

or

to always return an array of vrps

Reply:
I could keep TestHelper.load_vrp function but simply call TestHelper.load_vrps in it and just keep the first one. What do you think ? Something like : 

```
  def self.load_vrp(test, options = {})
    load_vrp(test, options)[0]
  end
```

Reply:
I agree !